### ISSUE TITLE
Linter support?

### ISSUE THREAD

**Issue by mstykow** [kentcdodds/use-deep-compare-effect#44]
- `use-deep-compare-effect` version: 1.6.1

Hi! Thanks for the great tool. Just wondering what happened to linter support for dependency array checks. I thought an older version had this but since of this newest version it somehow disappeared.

**Comment by kentcdodds** [comment]
I don't think it ever worked, but you can configure the linter rule to check this hook if you'd like

**Comment by dreadwail** [comment]
`react-hooks/exhaustive-deps` from [eslint-plugin-react-hooks](https://www.npmjs.com/package/eslint-plugin-react-hooks) comes with out-of-the-box support for applying to other custom hooks, so I believe you can do this and be good to go:

```json
{
  "rules": {
    "react-hooks/exhaustive-deps": ["warn", {
      "additionalHooks": "(useDeepCompareEffect)"
    }]
  }
}
```

Reference:

- https://www.npmjs.com/package/eslint-plugin-react-hooks
